components/needs_team/needs_team.jsx

```diff
@@ -17,44 +17,18 @@ import TeamStore from 'stores/team_store.jsx';
 import UserStore from 'stores/user_store.jsx';
 import BrowserStore from 'stores/browser_store';
 import * as GlobalActions from 'actions/global_actions.jsx';
-import Pluggable from 'plugins/pluggable';
 import Constants from 'utils/constants.jsx';
 import * as UserAgent from 'utils/user_agent.jsx';
 import * as Utils from 'utils/utils.jsx';
 import {loadProfilesForSidebar} from 'actions/user_actions.jsx';
 import {checkIfMFARequired} from 'utils/route';
-import AnnouncementBar from 'components/announcement_bar';
-import DeletePostModal from 'components/delete_post_modal.jsx';
-import EditPostModal from 'components/edit_post_modal';
-import GetPostLinkModal from 'components/get_post_link_modal';
-import GetTeamInviteLinkModal from 'components/get_team_invite_link_modal';
-import GetPublicLinkModal from 'components/get_public_link_modal';
-import InviteMemberModal from 'components/invite_member_modal.jsx';
-import LeaveTeamModal from 'components/leave_team_modal.jsx';
-import LeavePrivateChannelModal from 'components/modals/leave_private_channel_modal.jsx';
-import Navbar from 'components/navbar';
-import RemovedFromChannelModal from 'components/removed_from_channel_modal.jsx';
-import ResetStatusModal from 'components/reset_status_modal';
-import ShortcutsModal from 'components/shortcuts_modal.jsx';
-import SidebarRight from 'components/sidebar_right';
-import SidebarRightMenu from 'components/sidebar_right_menu';
-import TeamSettingsModal from 'components/team_settings_modal.jsx';
-import ImportThemeModal from 'components/user_settings/import_theme_modal.jsx';
-import UserSettingsModal from 'components/user_settings/user_settings_modal.jsx';
-import WebrtcNotification from 'components/webrtc/components/webrtc_notification.jsx';
-import WebrtcSidebar from 'components/webrtc/components/webrtc_sidebar.jsx';
-import ModalController from 'components/modal_controller';
-import TeamSidebar from 'components/team_sidebar';
-import Sidebar from 'components/sidebar';
-import ChannelView from 'components/channel_view';
-import PermalinkView from 'components/permalink_view';
-import MessageIdentifierRouter from 'components/message_indentifier_router';
 import {makeAsyncComponent} from 'components/async_load';
 import loadBackstageController from 'bundle-loader?lazy!components/backstage/backstage_controller';
 
 const BackstageController = makeAsyncComponent(loadBackstageController);
 
 import store from 'stores/redux_store.jsx';
+import ChannelController from '../channel_layout/channel_controller';
```

Comment:
Can you change this to an absolute import? We only really use relative imports when the file is in the same folder

Reply:
I am good with delaying (have to for joram's changes above anyway) and I will remove relative imports. I need to set my IDE better.